common/src/main/java/org/openmrs/analytics/FhirStoreUtil.java

```diff
@@ -158,4 +157,41 @@ private GoogleCredential getGoogleCredential() throws IOException {
 		return credential;
 	}
 	
+	public String executeRequest(HttpUriRequest request) {
```

Comment:
nit: Do you mind keeping this method where it was to reduce the amount of changes? In general, avoiding unnecessary changes will help tracking history of a line and code changes in future too. So unless it is refactoring, I prefer to avoid them (and for refactoring, pure refactoring commits are better).